### PR TITLE
125 change left sidebar

### PR DESCRIPTION
**Pull request by aurlic**

### CODE REVIEW

File: django/src/pages/templates/extends/base.html [hunk 1]
```diff
@@ -1,56 +1,173 @@
...
+								</svg>
+								<span>PONG</span>
+							</a>
+						</li>
+						<li>
+							<a href="/tictactoe">
+								<svg xmlns="http://www.w3.org/2000/svg" height="40px" viewBox="0 -960 960 960"
+									width="40px" fill="#000000">
+									<path
+										d="M186.67-120q-27 0-46.84-19.83Q120-159.67 120-186.67v-586.66q0-27 19.83-46.84Q159.67-840 186.67-840h586.66q27 0 46.84 19.83Q840-800.33 840-773.33v586.66q0 27-19.83 46.84Q800.33-120 773.33-120H186.67Zm0-66.67h151v-151h-151v151Zm217.66 0h151.34v-151H404.33v151Zm218 0h151v-151h-151v151ZM186.67-404.33h151v-151.34h-151v151.34Zm217.66 0h151.34v-151.34H404.33v151.34Zm218 0h151v-151.34h-151v151.34Zm-435.66-218h151v-151h-151v151Zm217.66 0h151.34v-151H404.33v151Zm218 0h151v-151h-151v151Z" />
```
Comment (Neffi42) [inline django/src/pages/templates/extends/base.html:88]: whats is that ?

**Review by Neffi42 — CHANGES_REQUESTED** [review]

File: nginx/src/css/sidebar.css [hunk 2]
```diff
@@ -1,37 +1,127 @@
...
 
-#sidebar-wrapper .sidebar-heading {
-	padding: 1rem;
-	font-size: 1.5rem;
+#sidebar>ul>li:first-child {
+	display: flex;
+	justify-content: flex-end;
+	margin-bottom: 16px;
+
+	.logo {
```
Comment (antoineverin) [inline nginx/src/css/sidebar.css:32]: not valid css

File: nginx/src/css/sidebar.css [hunk 3]
```diff
@@ -1,37 +1,127 @@
...
+		font-weight: 600;
+	}
 }
 
-#sidebar-wrapper .list-group-item {
-	padding: 1rem 1.5rem;
+/* #sidebar ul li.active a {
```
Comment (antoineverin) [inline nginx/src/css/sidebar.css:37]: remove comment

File: nginx/src/css/sidebar.css [hunk 4]
```diff
@@ -1,37 +1,127 @@
...
+#sidebar .dropdown-btn:hover {
+	background-color: var(--hover-clr);
+}
+
+#sidebar .sub-menu {
+	display: grid;
+	grid-template-rows: 0fr;
+	transition: 300ms ease-in-out;
+
+	>div {
```
Comment (antoineverin) [inline nginx/src/css/sidebar.css:86]: not valid css

File: nginx/src/css/sidebar.css [hunk 5]
```diff
@@ -1,37 +1,127 @@
...
+
+#toggle-btn {
+	margin-left: auto;
+	padding: 1em;
+	border: none;
+	border-radius: .5em;
+	background: none;
+	cursor: pointer;
+
+	svg {
```
Comment (antoineverin) [inline nginx/src/css/sidebar.css:115]: not valid css

**Review by antoineverin — CHANGES_REQUESTED** [review]
remove svgs in html and px units

File: django/src/pages/templates/extends/base.html (hunk 1 above)
Comment (aurlic) [reply django/src/pages/templates/extends/base.html]: SVG path for defining icons
Changed in commit : [d24d348](https://github.com/TheSecretOrganization/ft_transcendence/pull/201/commits/d24d34875c8598f849f6a6c2f6e5f2b83ab06290)

File: nginx/src/css/sidebar.css (hunk 2 above)
Comment (aurlic) [reply nginx/src/css/sidebar.css]: Addressed in : [d24d348](https://github.com/TheSecretOrganization/ft_transcendence/pull/201/commits/d24d34875c8598f849f6a6c2f6e5f2b83ab06290)

File: nginx/src/css/sidebar.css (hunk 4 above)
Comment (aurlic) [reply nginx/src/css/sidebar.css]: adressed in : [85e156a](https://github.com/TheSecretOrganization/ft_transcendence/pull/201/commits/85e156af24302898ff928ba7a46c4a1a9e8c5210)

**Review by Neffi42 — APPROVED** [review]

**Review by antoineverin — APPROVED** [review]

**Pull request merged**